### ISSUE TITLE
Eierlegende Wollmilchsau - inkl. erste EK

### ISSUE THREAD

**Issue by csalzmann** [FNSKtZH/apflora#121]
Für effiziente Zielauswertung der APs bräuchte es neben der letzten auch die erste EK - denn die ursprünglichen Populationen müssen meist um einen Faktor x vergrössert werden.

**Comment by barbalex** [comment]
hi hi - die Sau produziert jetzt auch noch Seide :-)

Im Ernst: Ob der Server das erträgt, muss sich herausstellen. Diese Änderung wird aufwändig und es gibt ein verhältnismässig grosses Risiko, dass wir merken, dass wir den Server (für die Abfrage für nur eine Art) bzw. meinen PC (für die Abfrage für alle Arten) überfordern.

Wollt ihr, dass ich ca. 2 Stunden investiere um das zu testen?

**Comment by barbalex** [comment]
siehe auch: https://github.com/FNSKtZH/apflora/issues/24

**Comment by barbalex** [comment]
Die eierlegende Wollmilchsau liefert heute 94 Felder (für 10'255 Datensätze in einer 2.1MB grossen Datei).
Davon stammen 44 Felder aus der Kontrolle.

Wenn ich nun für die erste Kontrolle nochmals 44 Felder ergänze, haben wir danach Zeilen mit 138 Feldern bzw. Spalten.

Ausserdem muss ich bei 88 Feldern die Feldnamen ergänzen mit : "Erste Kontrolle:" bzw. "Letzte Kontrolle:". Die Feldbreiten werden also zunehmen.

Ich gehe davon aus, dass niemand mehr diesen Export sinvoll nutzen kann, ohne zuerst ziemlich radikal Felder auf die für den jeweiligen Anwendungsfall zu nutzenden zu reduzieren.

Meine Frage ist daher: Ist es sinnvoll, wenn ich im bestehenden Export diese 44 Felder ergänze? Oder gibt es eine bessere Lösung?

**Comment by barbalex** [comment]
Zweite Frage: Ihr wollt die erste Kontrolle nutzen,

> um Zuwachs in autochthonen Populationen gegenüber Anfangszustand zu ermitteln

Das heisst aber auch: Ich muss nicht nur die erste Kontrolle, sondern auch ihre Zählungen ergänzen. Richtig?

Und wenn ich das mache: Ist euch bewusst

- dass die Zählungen, weil es diverse Einheiten und Methoden gibt, nicht automatisch ausgewertet werden können? (ausser ich soll im Export nur die Zählungen einer bestimmten Einheit kombiniert mit einer bestimmten Methode filtern)
- dass die Zählungen, weil mehrere möglich sind, kommagetrennt aufgelistet sind?
![zaehlungen](https://user-images.githubusercontent.com/669783/31073026-836066c4-a76a-11e7-8787-f04f4bee49e1.png)

**Comment by csalzmann** [comment]
ja, wir brauchen die Zählungen. Kommagetrennt wär OK, automatische Auswertung ist sicher nicht möglich, weil einfach zu oft versch Einheiten benützt werden.

Es bräuchte im Export aber wscheinlich nicht so viele Felder, da könnte man radikal kürzen, wenn es eben spezifisch um den Vergleich ginge. Wäre das vielleicht ein Weg, die Sache zu verschlanken? 

Von meinem iPhone gesendet

**Comment by barbalex** [comment]
Kommt drauf an, wozu dieser Export genutzt wird. Das musst du sagen. Du hast diese Möglichkeiten:

- alle Felder wünschen (und dann jeweils im Export viele ausblenden)
- die Felder auflisten, welche ihr bei diesem Export braucht
- für diesen Zweck einen zusätzlichen Export wünschen und dafür die Felder einschränken. Wird aber aufwändig(er), weil bei diesem Export pro Art exportiert wird

**Comment by csalzmann** [comment]
Felder auflisten! Habe die Felder im Wollmichsau-Export gelb hinterlegt, die ich haben möchte. Dazu kämen analog zur letzten EK diejenigen der 1. EK.
Pro Art wäre OK falls das was nützt.
[anzkontrinklletzterundletztertpopber_2016_Attributauswahl.xlsx](https://github.com/FNSKtZH/apflora/files/1474917/anzkontrinklletzterundletztertpopber_2016_Attributauswahl.xlsx)

**Comment by barbalex** [comment]
super, danke
